src/dotty/tools/dotc/FromTasty.scala

```diff
@@ -64,6 +64,9 @@ object FromTasty extends Driver {
   }
 
   class ReadTastyTreesFromClasses extends FrontEnd {
+
+    override def isTyper = false
```

Comment:
Unfortunately, this is not enough. 
A lot of code checks for `Phases.isAfterTyper` that is entirely independent from this method